rosbag2_transport/src/rosbag2_transport/recorder.cpp

```diff
@@ -381,6 +385,62 @@ std::string Recorder::serialized_offered_qos_profiles_for_topic(const std::strin
   return YAML::Dump(offered_qos_profiles);
 }
 
+std::string type_hash_to_string(const rosidl_type_hash_t & type_hash)
+{
+  if (type_hash.version != 1) {
+    // version is either unset or a later version we don't know how to serialize.
+    return "";
+  }
+  rcutils_allocator_t allocator = rcutils_get_default_allocator();
+  char * stringified_type_hash = NULL;
```

Comment:
Please use `nullptr` in C++ code
```suggestion
  char * stringified_type_hash = nullptr;
```

Reply:
yeah - wasn't sure since it's calling into C.